docs/reference/modules/admin-bar.md

```diff
@@ -0,0 +1,125 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/admin-bar`
+
+**Alias:** `apos.adminBar`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+This module implements Apostrophe's admin bar at the top of the page. Any module can register a button (or more than one) for the bar by calling this module's `add` method. Buttons can also be grouped into dropdown menus and restricted to those with particular permissions. The [apostrophe-pieces](/reference/modules/piece-type.md) takes advantage of this module automatically.
```

Comment:
The apostrophe-pieces *module* takes advantage...

Reply:
`apostrophe-pieces` isn't a thing in A3, you mean to to say `@apostrophecms/piece-type`